network.c

```diff
@@ -553,13 +609,15 @@ static ssize_t network_write_data(struct iiod_client_pdata *io_ctx,
 				  const char *src, size_t len,
 				  unsigned int timeout_ms)
 {
-	return network_send(io_ctx, src, len, MSG_NOSIGNAL, timeout_ms);
+	return (ssize_t)translate_error_for_api(
+		(int)network_send(io_ctx, src, len, MSG_NOSIGNAL, timeout_ms));
```

Comment:
Do we need the cast at all? In linux for sure we don't but maybe windows complains about it